docs/src/global/breakpointActions.js

```diff
@@ -1,11 +1,10 @@
-'use strict';
 
-import { appReduxStore } from '../global/configureReduxStore.js';
+import { appReduxStore } from './configureReduxStore.js';
 
 export default class BreakpointActions {
     static update() {
-        const clientHeight = document.documentElement.clientHeight;
-        const clientWidth = document.documentElement.clientWidth;
+        const { clientHeight } = document.documentElement;
+        const { clientWidth } = document.documentElement;
```

Comment:
Wouldn't we want `const { clientHeight, clientWidth } = document.documentElement;`?
Probably ESLINT auto fix is not quite smart enough to have been able to do that. :-)

Reply:
Must not be smart enough. :/